examples/src/main/scala/org/apache/spark/examples/ml/DecisionTreeExample.scala

```diff
@@ -167,17 +167,17 @@ object DecisionTreeExample {
       testInput: String,
       algo: String,
       fracTest: Double): (DataFrame, DataFrame) = {
-    val sqlContext = new SQLContext(sc)
+    val spark = SparkSession.builder.getOrCreate()
```

Comment:
I wonder if we should have an internal `SparkSession.builder.create(sc)`. It's kind of weird to just ignore the argument. That's a separate issue though.